test-lib.sh

```diff
@@ -538,8 +577,14 @@ EOF
     else
         echo "CMD /usr/libexec/s2i/run" >>"$df_name"
     fi
+
+    if $ca_file; then
+        # Check if -v parameter is present in s2i_args and add it into docker build command
+        mount_options=$(echo "$s2i_args" | grep -o -e '\(-v\)[[:space:]]\.*\S*')
```

Comment:
@pkubatrh In case `ca_file` is present and NPM_REGISTRY variable was not specified, then `grep` command failed, because of no option like `-v` in `s2i_args` is present. There are two possibilities from my point of view.
before command grep add instruction `set -x` and afterward `set -e` back again or wait till internal PR is merged.

Any other idea?

Reply:
How about adding `|| true` to the grep command? That way the result value will never cause the script to fail (and we do not care about it anyway).